src-opam/distro.ml

```diff
@@ -1093,16 +1093,16 @@ let latest_tag_of_distro (t : t) =
   tag_of_distro latest
 
 type package_manager =
-  [ `Apt | `Yum | `Apk | `Zypper | `Pacman | `Cygwin | `Windows ]
+  [ `Apt | `Dnf | `Apk | `Zypper | `Pacman | `Cygwin | `Windows ]
 [@@deriving sexp]
 
 let package_manager (t : t) =
   match t with
   | `Ubuntu _ -> `Apt
   | `Debian _ -> `Apt
-  | `CentOS _ -> `Yum
-  | `Fedora _ -> `Yum
-  | `OracleLinux _ -> `Yum
+  | `CentOS _ -> `Dnf
```

Comment:
Not ALL CentOS supports DNF. CentOS 7 (which is what I'm currently using the docker containers for) does not have DNF, but just Yum, so might be useful to retain 'Yum' as a package manager and just add DNF.
(Although Yum is still present as an alias in newer versions of Fedora, although unsure for how long)